Support_Your_Locals/Models/ViewModels/BusinessRegisterModel.cs

```diff
@@ -6,17 +6,17 @@ namespace Support_Your_Locals.Models.ViewModels
     public class BusinessRegisterModel
     {
 
-        [Required(ErrorMessage = "Please add your business description")]
+        [Required(ErrorMessage = "Please add description for your business")]
         public string Description {get; set;}
-        [RegularExpression(@"^[+]*[(]{0,1}[0-9]{1,4}[)]{0,1}[-\s\./0-9]*$", ErrorMessage = "Bad phone number")]
-        [Required(ErrorMessage = "Please enter your business phone number")]
+        [RegularExpression(@"^[+]*[(]{0,1}[0-9]{1,4}[)]{0,1}[-\s\./0-9]*$", ErrorMessage = "Entered phone number is not correct")]
+        [Required(ErrorMessage = "Please enter correct phone number for your business")]
         public string PhoneNumber {get; set;}
-        [Required(ErrorMessage = "Please enter your business header")]
+        [Required(ErrorMessage = "Please enter header for your business")]
         public string Header {get; set;}
         public string Longitude { get; set; }
         public string Latitude { get; set; }
         public string Picture { get; set; }
-        public TimeSheetRegisterViewModel[] Workdays { get; set; } = new TimeSheetRegisterViewModel[7];
-        public List<Product> Products { get; set; } = new List<Product>();
+        public List<TimeSheetRegisterViewModel> Workdays { get; set; } = new List<TimeSheetRegisterViewModel>();
```

Comment:
This should be array of size 7 because if not only edit will work (empty list wll not be rendered on add advertisement).